cli/spec/support/output_helpers.rb

```diff
@@ -52,23 +54,25 @@ module OutputHelpers
         line = 0
         @real.zip(@expected) do |real, expected|
           line += 1
-          error = values_match?(real.split(/\S+/), expected)
```

Comment:
@kke I don't think `values_match?` was actually returning an error String here... not 100% sure, but I think it returns a "good/bad" boolean...?

Also, the `split(/\S+/)` also resulted in an Array containing only whitespace strings:

```
       line 1: expected [":warning node-1", "1.1-dev", "online", "1 / 3", "-"], received ["", " ", "                      ", "         "]
       line 2: expected [":warning node-2", "1.1-dev", "online", "2 / 3", "-"], received ["", " ", "           ", "            ", " ", " ", "     "]
       line 3: expected [":offline (initializing)", "", "offline", "-", "-"], received ["", " ", "           ", "            ", " ", " ", "     "]
```